### ISSUE TITLE
Issue with openjdk8 runtime when setting tool path

### ISSUE THREAD

**Issue by chrisbenincasa** [aws/aws-codebuild-docker-images#289]
I built the ubuntu standard image 3.0 at `413f77ee5474b3219eff86962dddf07e284c9f5b` . When I run a simple build (below) I get an error at the runtime setup phase.

buildspec
```
version: 0.2

phases:
  install:
    runtime-versions:
      java: openjdk8
    commands:
      - which java
      - echo Entered the install phase...
  build:
    commands:
      - cd server
```

Console output when running locally:
```
agent_1  | [Container] 2019/12/20 15:47:33 Running command echo "Installing Java version 8 ..."
agent_1  | Installing Java version 8 ...
agent_1  |
agent_1  | [Container] 2019/12/20 15:47:33 Running command export JAVA_HOME="$JAVA_8_HOME"
agent_1  |
agent_1  | [Container] 2019/12/20 15:47:33 Running command export JRE_HOME="$JRE_8_HOME"
agent_1  |
agent_1  | [Container] 2019/12/20 15:47:33 Running command export JDK_HOME="$JDK_8_HOME"
agent_1  |
agent_1  | [Container] 2019/12/20 15:47:33 Running command for tool_path in "$JAVA_8_HOME"/bin/* "$JRE_8_HOME"/bin/*;
agent_1  |  do tool=`basename "$tool_path"`;
agent_1  |   if [ $tool != 'java-rmi.cgi' ];
agent_1  |   then
agent_1  |   update-alternatives --list "$tool" | grep -q "$tool_path" \
agent_1  |    && update-alternatives --set "$tool" "$tool_path";
agent_1  |   fi;
agent_1  | done
agent_1  | /codebuild/output/tmp/script.sh: 6: [: LICENSE: unexpected operator
agent_1  | /codebuild/output/tmp/script.sh: 6: [: LICENSE: unexpected operator
agent_1  |
```

**Comment by aditynar** [comment]
Hi, this is Aditya from CodeBuild. Could you provide some more information on your build environment?

Did you select the latest version of the Ubuntu Standard version 3.0 when running this build?

**Comment by chrisbenincasa** [comment]
Sorry for the delay. What additional information do you need? I am building the Ubuntu Standard version 3.0 (as noted in the issue). Can this be reopened? This is still an issue I am experiencing and is totally blocking any builds I attempt.

**Comment by subinataws** [comment]
Did you use the Ubuntu standard 3.0 image as provided by CodeBuild or did you make any modifications to the Dockerfile or runtimes.yml file before running this buildspec? We couldn't repro this issue on our end, so do provide more information on the setup.

**Comment by chrisbenincasa** [comment]
I am running the build locally and built the aws/codebuild/standard:3.0 image from a checkout of this repo following the instructions in the README. I made no modifications to any of the code in the repo. I can try again with a more fresh checkout.

**Comment by chrisbenincasa** [comment]
This issue must've gotten fixed between `413f77ee5474b3219eff86962dddf07e284c9f5b` and HEAD. I rebuild the image using a fresh checkout of the repo and the build seems to be working now!
